template/mkdocs.yml.jinja

```
@@ -0,0 +1,28 @@
+site_name: {{ project_name }}
+site_description: {{ project_description }}
+site_url: {{ documentation_url }}
+repo_url: {{ project_url }}
+repo_name: {{ project_url.split('/')[-2:] | join('/') }}
+
+theme:
+  name: material
+
+nav:
+  - Home: index.md
+{%- if project_type == 'package' %}
+  - Reference: reference/
+{%- endif %}
```

Would add a watch on `src` if there's a project reference.
```suggestion
nav:
  - Home: index.md
{%- if project_type == 'package' %}
  - Reference: reference/

watch:
  - src
{%- endif %}
```